scripts/bootstrap.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# The below ` || exit 1;` is added to stop the script if a command fails
```

Comment:
You can do `set -e` for this exact behavior!

I also like including `-x` (`set -ex`), which prints each command before it runs.

Reply:
Do you mean `set -ex insert_command_here`?

Reply:
Nope, that's the whole line.

```bash
#!/usr/bin/env/bash

set -ex

npm run lerna:bootstrap
# ...
```

Reply:
oh cool, thanks! sorry for asking instead of googling :sweat_smile: